pwa/src/translations/en.ts

```diff
@@ -69,4 +69,10 @@ export const en = {
   "Go back": "Go back",
   Version: "Version",
   "Management organization": "Management organization",
+  User: "User",
+  "My catalogi": "My catalogi",
```

Comment:
We've been using "Open Catalogi", therefore I'd suggest making this "My Catalogi"